internal/container/docker.go

```diff
@@ -25,17 +25,20 @@ import (
 	"strings"
 	"time"
 
+	"github.com/Masterminds/semver/v3"
 	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/log"
 )
 
 var ErrDockerNotFound = fmt.Errorf("%w: docker not found in your system, check requirements at https://dochub.mongodb.org/core/atlas-cli-deploy-local-reqs", ErrContainerEngineNotFound)
+var ErrDeterminingDockerVersion = errors.New("could not determine docker version")
 var errParseHealthCheck = errors.New("parsing image healthcheck failed")
 var errListContainer = errors.New("container listing failed")
 var errParsingContainer = errors.New("container parsing failed")
 var errDecodingJSON = errors.New("container decoding failed")
 var errParsingPorts = errors.New("parsing ports failed")
 var errConvertHostPort = errors.New("converting host port failed")
 var errConvertContainerPort = errors.New("converting container port failed")
+var minDockerVersion = semver.New(27, 0, 0, "", "") //nolint:mnd
```

Comment:
Our promise is the last two, so I would stick 26

Reply:
`27.0+` is what the docs say. Should we update the docs too?

Reply:
https://www.mongodb.com/docs/atlas/cli/current/atlas-cli-deploy-local/

Reply:
yes our current epic (linux stability) will update the docs

Reply:
dicussed offline, let's keep only latest